__tests__/data/iterable_source_input.json

```diff
@@ -0,0 +1,16 @@
+[
+  {
```

Comment:
We need to add test cases for all the types of events as share in research spec

Reply:
I will add all the test cases in the next commit.